### PR TITLE
CONTRIBUTING.md: Fix wording

### PR DESCRIPTION
**Pull request by felixge**
Semver releases are identified via `MAJOR.MINOR.PATCH` syntax.

grpc is known to break APIs that they consider experimental in MINOR releases.

### CODE REVIEW

File: CONTRIBUTING.md
Comment (Julio-Guerra) [inline CONTRIBUTING.md:43]:
```suggestion
When adding a new dependency, especially for `contrib/` packages, prefer the minimum secure versions of any modules rather than the latest versions. This is to avoid forcing upgrades on downstream users for modules such as `google.golang.org/grpc` which often introduces breaking changes within minor versions.
```

**Review by Julio-Guerra — APPROVED** [review]

**Comment by felixge** [comment]
Thanks @Julio-Guerra – despite this being non-load bearing, I'll wait until the release freeze is over before merging.

**Review by nsrip-dd — APPROVED** [review]

**Pull request merged**
